src/main/java/net/coderbot/iris/shaderpack/IdMap.java

```diff
@@ -59,20 +60,47 @@
 		// TODO: Properly override block render layers
 	}
 
+	/**
+	 * Loads properties from a properties file in a shaderpack path
+	 */
 	private static Optional<Properties> loadProperties(Path shaderPath, String name) {
-		Properties properties = new Properties();
+		if (shaderPath == null) return Optional.empty();
```

Comment:
`if` statements should always be surrounded with braces. Also, why is this null check here in the first place? `shaderPath` should never be null.